### PR TITLE
PyNE

### PR DESCRIPTION
**Pull request by scopatz**
This is a PyNE package attempt. Still some cleanup to do, but I want to make sure that it works on the CI systems.

### CODE REVIEW

**Comment by scopatz** [comment]
Hmm anyone on @conda-forge/core know about this error message on linux:

```
...Traceback (most recent call last):
  File "/opt/conda/lib/python3.5/site-packages/conda_build_all/builder.py", line 200, in build
    conda_build.api.build(meta.meta, config=config)
AttributeError: module 'conda_build' has no attribute 'api'
```

Note that this recipe builds just fine for me on Ubuntu.

**Comment by mwcraig** [comment]
That message is because conda-build-all 1.0.0 assumes conda-build >=2 (which has an api module), but the builds here are still pinned at conda-build=1.21.* (which doesn't), though maybe it is https://github.com/SciTools/conda-build-all/issues/62 instead.

**Comment by scopatz** [comment]
Is it the gcc package that is causing this?

**Comment by scopatz** [comment]
Thanks for taking a look @mwcraig!

**Comment by scopatz** [comment]
Ok this package also works now!  A merge would be most appreciated! Thanks!

File: recipes/pyne/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# note that builds timeout on mac with many versions of python & numpy
+#
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - pkg-config
+    - setuptools
+    - gcc
```
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: So we have `gcc` and the `toolchain`? Any reason we can't just use `toolchain`?

Comment (scopatz) [inline recipes/pyne/meta.yaml]: Fortran, right? You need the gcc package to get gfortran I thought.

File: recipes/pyne/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    - libgcc
+    - libgfortran
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
```
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: `0.2.18|0.2.18.*`

File: recipes/pyne/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    - xz 5.2.*
+    - moab
+    - cython
+    - numpy x.x  # [linux]
+    - numpy 1.11.*  # [osx]
+    - pytables
+  run:
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
```
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: `0.2.18|0.2.18.*`

File: recipes/pyne/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - moab
+    - cython
+    - numpy x.x  # [linux]
```
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: This shouldn't have a selector at all.

File: recipes/pyne/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  run:
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - moab
+    - cython
```
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: Is `cython` needed at run time?

File: recipes/pyne/meta.yaml (hunk 1 above)
Comment (scopatz) [inline recipes/pyne/meta.yaml]: Also, I had https://github.com/conda-forge/hdf5-feedstock/pull/45 accepted yesterday.

File: recipes/pyne/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
+{% set version = "0.5.1" %}
+{% set org = "pyne" %}
+{% set branch = "0.5.1" %}
```
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: Why have org/branch?

File: recipes/pyne/meta.yaml (hunk 4 above)
Comment (scopatz) [inline recipes/pyne/meta.yaml]: See comment above.  The Travis resources are too slow to compile more than 2 versions on Mac without timing out.

Comment (scopatz) [inline recipes/pyne/meta.yaml]: If you know a better way to do this, I am all ears.  Do you think it should go into the build-skip

File: recipes/pyne/meta.yaml (hunk 5 above)
Comment (scopatz) [inline recipes/pyne/meta.yaml]: Whoops no, but this is in the build block.  I remove the one below.

**Comment by scopatz** [comment]
Thanks for the review @jakirkham. I have made the requested changes.  Any suggestions on the selector?  Or do you think it is OK considering the timeout issue on Travis?

**Comment by scopatz** [comment]
OK, this is passing again!

**Comment by scopatz** [comment]
@jakirkham and @conda-forge/core is there anything else that needed to happen here?

**Comment by ocefpaf** [comment]
Let's get this into a feedstockand iterate there.

**Comment by scopatz** [comment]
Thanks @ocefpaf!

File: recipes/pyne/meta.yaml (hunk 5 above)
Comment (jakirkham) [inline recipes/pyne/meta.yaml]: Yeah, sometimes GitHub moves the comments around. I never no why. 😒

**Pull request merged**